x/protorev/keeper/epoch_hook_test.go

```diff
@@ -162,7 +162,7 @@ func (s *KeeperTestSuite) TestUpdateHighestLiquidityPools() {
 			},
 			expectedBaseDenomPools: map[string]map[string]keeper.LiquidityPoolStruct{
 				"epochTwo": {
-					"uosmo": {Liquidity: sdk.NewInt(2000000), PoolId: 49},
+					"uosmo": {Liquidity: sdk.Int(sdk.NewUintFromString("999999999000000001000000000000000000")), PoolId: 49},
```

Comment:
Yes, but the Uint implementation is actually a big.Int lol: https://github.com/osmosis-labs/cosmos-sdk/blob/fbf8ba8a41fe455ebea7785e9e4fbac750b8dc3b/types/uint.go#L12

The reason this change happened is because I use the test suite way to create a new CL pool based on another Osmosis team member recommendation, and then the liquidity levels created with that function changed to be large amounts, thus making the liquidity Mul this large: https://github.com/osmosis-labs/osmosis/blob/6c744ac96782041fdcc08f4993c53494d8a5fa38/x/protorev/keeper/keeper_test.go#L894

Reply:
how about just sdk.NewIntFromString then? 

i believe it uses bigInt too

Reply:
Just checked, NewIntFromString returns an sdk.Int and a bool (instead of just sdk.Uint value which can be converted inline), so wouldn't cleanly fit into the params for the test.

If no strong reason to convert, I think we should just keep as is.

Reply:
sounds good